homeassistant/components/lamarzocco/coordinator.py

```diff
@@ -149,14 +97,44 @@ async def _async_handle_request(
             _LOGGER.debug(ex, exc_info=True)
             raise UpdateFailed("Querying API failed. Error: %s" % ex) from ex
 
-    def async_get_ble_device(self) -> BLEDevice | None:
-        """Get a Bleak Client for the machine."""
-        # according to HA best practices, we should not reuse the same client
-        # get a new BLE device from hass and init a new Bleak Client with it
-        if not self._use_bluetooth:
-            return None
 
-        return async_ble_device_from_address(
-            self.hass,
-            self.lm.lm_bluetooth.address,
-        )
+class LaMarzoccoMachineUpdateCoordinator(
+    LaMarzoccoUpdateCoordinator[LaMarzoccoMachine]
+):
```

Comment:
We only have 1 implementation of the `LaMarzoccoUpdateCoordinator`, why is it a generic?

Reply:
To prepare for the grinder which will come in a later PR

Reply:
Can we scope this PR to make sure the integration can get working again?